### ISSUE TITLE
Consider implementing MD045 no-alt-text "Images should have alternate text (alt text)"

### ISSUE THREAD

**Issue by DavidAnson** [markdownlint/markdownlint#187]
Please see https://github.com/DavidAnson/markdownlint/commit/29e64d86dd14a3b4f341ce9648f0f675aa486591 for context.
Please reserve MD045 for this rule regardless.

**Comment by shahzebsiddiqui** [comment]
When is this rule going to be added? I dont see it in https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md
